index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+
+const db = require('./dbConfig.js');
+
+const server = express();
+
+server.use(express.json());
+server.use(helmet());
+server.use(cors());
+
+// endpoints
+
+server.get('/', (req, res) => {
+    res.send('Abracdabra!');
+});
+
+server.post('/api/register', (req, res) => {
+    const creds = req.body;
+    const hash = bcrypt.hashSync(creds.password, 10);
+    creds.password = hash;
+
+    db('users')
+    .insert(creds)
+    .then(ids => {
+        const id = ids[0];
+
+        res.status(201).json(id);
+    })
+    .catch(err => res.status(500).send(err))
```

Comment:
It's a good idea to `console.log` in your catch blocks, for you often get better error messages there than what is received by PostMan.

```js
// continued from db('users').insert(creds)
.catch(err => {
    console.log('/api/register POST ERROR:', err);
    res.status(500).send(err);
}
```